.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+## Description
+
+> Provide a brief description about **why** this PR is necessary. Be sure to provide context.
+
+## Related issues and/or PRs
+
+> If this PR addresses or references any issues and/or other PRs, list them here.
+
+## Changes made
+
+> Outline the specific changes made in this pull request. Include relevant details, such as added features, bug fixes, code refactoring, or improvements.
+
+## Checklist
+
+> The following is a best-effort checklist. If any items in this checklist are not applicable to this PR or are dependent on other, unmerged PRs, please still mark the checkboxes after you have read and understood each item.
+
+- [ ] I have commented my code, particularly in hard-to-understand areas.
+- [ ] I have updated the documentation to reflect the changes.
+- [ ] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
+- [ ] Tests (unit, integration, etc.) have been added for the changes.
+- [ ] My changes generate no new warnings.
+- [ ] Any dependent changes in other PRs have been merged and published.
+
+## Additional notes (optional)
+
+> Provide any additional information or notes that may be relevant to the reviewers or stakeholders.
+
+## Release notes
+
+> Provide release note text for this PR based on the guidelines at [Guidelines for Writing Release Notes](https://developers.scalar-labs.com/docs/style-guide/release-notes/).
+> 
+> For example:
+> 
+> Added validation for the Consensus Commit mutation operation.
```

Comment:
I don't think we need release nodes for this repository.

Reply:
Thanks for confirming! I've removed the `## Release notes` section.